ui/webui/test/check-basic

```diff
@@ -139,6 +139,7 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
         b.wait_visible("#critical-error-modal")
 
         # Test review of logs
+        self.addCleanup(m.execute, "rm -f /tmp/webui.log")
```

Comment:
I didn't clean up the log file because this is a destructive test, but it may be better to clean up anyway